docs/source/manual/auth.rst

```diff
@@ -280,12 +280,12 @@ Add this dependency into your ``pom.xml`` file:
 OAuth Example
 -------------
 
-When you build your ``ResourceTestRule``, add the ``GrizzlyWebTestContainerFactory`` line.
+When you build your ``ResourceExtension``, add the ``GrizzlyWebTestContainerFactory`` line.
 
 .. code-block:: java
 
     @Rule
```

Comment:
Do these @Rule and @ClassRule annotations need to be replaced?

Reply:
Good catch.  I cleaned up the old annotations and added the ExtendsWith annotation to the test class.